### PR TITLE
wip: start scan results page

### PR DESCRIPTION
**Pull request by groovecoder**
This is just a starting point for @nhnt11 to fill in the code described in the comment of `routes/home.js`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "client-sessions": "^0.8.0",
     "dotenv": "^5.0.1",
     "express": "^4.16.2",
+    "express-hbs": "^1.0.4",
```
Comment (pdehaan) [inline package.json]: Do we still need standalone `hbs` below if we're including `express-hbs`?

Comment (groovecoder) [reply package.json]: Probably not.

File: views/scan.hbs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <h2>Uh oh! Here are your results</h2>
+        <h3>{{ email }} <a href="">Subscribe</a></h3>
+        <div class="callout alert"><img src="http://via.placeholder.com/100x100">Really bad breach</div>
+        <div class="callout warning"><img src="http://via.placeholder.com/100x100">Bad breach</div>
+        <div class="callout secondary"><img src="http://via.placeholder.com/100x100">Regular breach</div>
+      </div>
+
+      <div class="cell small-4">
+        <p>Title that encourages user to try again</p>
+        <form action="/scan" method="post">
```
Comment (pdehaan) [inline views/scan.hbs]: This `<form>` is duplicated (yet subtly different) from https://github.com/mozilla/blurts-server/pull/73/files#diff-47ab88469c3f63d8b827d9130935ae0fR21

Thoughts on making this a shared HBS partial? Conversely, I can submit a followup bug after this is merged to DRY the templates. ¯\\\_(ツ)\_/¯

File: views/scan.hbs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <h1><a href="/">Firefox Breach Alerts</a></h1>
+      </div>
+      <div class="top-bar-right">
+        <ul class="menu">
+          <li><a href="/">About</a></li>
+          <li><a href="#tips">Tips</a></li>
+          <li><a class="button" data-open="subscribe-modal">Subscribe</a></li>
+        </ul>
+      </div>
+    </div>
```
Comment (pdehaan) [inline views/scan.hbs]: Similarly, this block seems the same as https://github.com/mozilla/blurts-server/pull/73/files#diff-47ab88469c3f63d8b827d9130935ae0fR2

Not sure if we should move into "default.hbs" template, or into a reusable partial.

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/blurts-server/pull/74

**Pull request closed without merging**